src/extlibs/hrvo/hrvo_agent.cpp

```diff
@@ -474,16 +483,17 @@ void HRVOAgent::computePreferredVelocity()
             // Calculate the maximum velocity towards the preferred velocity, given the
             // acceleration constraint
             pref_velocity_ =
-                velocity_ + (max_accel_ * simulator_->getTimeStep()) * (dv / abs(dv));
+                velocity_ + (max_accel_ * simulator_->getTimeStep()) * (dv / dv.length());
```

Comment:
pretty sure this works
```suggestion
                velocity_ + (max_accel_ * simulator_->getTimeStep()) * (dv.normalize());
```